src/main/java/frc/robot/commands/ArmJoystickConeCommand.java

```diff
@@ -87,7 +92,18 @@ public void initialize() {
     m_timer2.stop();
     m_timer2.reset();
 
+    topBrakeTimer.stop();
+    topBrakeTimer.reset();
+    botBrakeTimer.stop();
+    botBrakeTimer.reset();
+
     z_offset = 0;
+
+    m_topDebounce.calculate(false);
+    m_topDebounce.calculate(false);
```

Comment:
Why this is called twice?
Is this for m_bottomDebounce?

Reply:
good catch! it should be fixed and pushed now